### PR TITLE
Add quantity to books.

### LINKED ISSUE #6: Store quantity to books

### PR DESCRIPTION
**Pull request by dmitryrck**
Closes #6

### CODE REVIEW

File: spec/features/books_spec.rb [hunk 1]
```diff
@@ -24,10 +24,13 @@
       fill_in 'ISBN', with: '9781118084786'
       fill_in 'Title', with: 'Ruby on Rails For Dummies'
       fill_in 'Author(s)', with: 'Barry Burd'
+      fill_in 'Quantity', with: '10'
```
Comment (dmitryrck) [inline spec/features/books_spec.rb:27]: The next PR I will add tests to `image_link` :/

**Comment by dmitryrck** [comment]
@rafaelrpbelo Cloud you review and merge if it is ok? ;)

File: spec/features/books_spec.rb [hunk 2]
```diff
@@ -42,6 +45,8 @@
         click_on 'Criar'
 
         expect(page).to have_content 'Ruby on Rails For Dummies'
+
+        expect(Book.last.quantity).to eq 0
```
Comment (rafaelrpbelo) [inline spec/features/books_spec.rb]: we could create with ISBN and quantity too...

Comment (dmitryrck) [inline spec/features/books_spec.rb]: Let me see :)

Comment (dmitryrck) [inline spec/features/books_spec.rb]: @rafaelrpbelo √

**Comment by dmitryrck** [comment]
Thanks.

**Pull request merged**